source/code/source/playfab/PlayFabCurlHttpPlugin.cpp

```diff
@@ -115,12 +115,13 @@ namespace PlayFab
     constexpr char requestIdHeaderKey[] = "X-RequestId: ";
```

Comment:
The HTTP header spec allows for 0 or more linear whitespace after the colon, which are not considered part of the value. The value may also be followed by linear whitespace, which is not part of the value.
https://tools.ietf.org/html/rfc2616#section-4.2

Reply:
Since I'm already dropping the final requestId into a std::string, I can piggyback on that, add all the whitespace chars to that as well, and then use the std::string interface to remove them.